pkgs/data/fonts/ttf-indic/default.nix

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Move this file to `pkgs/by-name/tt/ttf-indic/package.nix`.

Reply:
done.